Document/0x05b-Basic-Security_Testing.md

```diff
@@ -752,15 +752,44 @@ Check the [Termux Wiki](https://wiki.termux.com/wiki/Remote_Access "Termux Remot
 
 #### Obtaining and Extracting Apps
 
--- ToDo: <https://github.com/OWASP/owasp-mstg/issues/1238>
+There are several ways of extracting apk files from a device. You will need to decide which one is the easiest method depending if the app is public or private.
 
-##### App Store
+##### Alternative App Stores
 
-##### Recovering the App Package from the Device
+One of the easiest option is to download the apk from websites that mirror public applications from the Google Play Store. However, keep in mind that these sites are not offical and there is no guarantee that the application hasn't been repackaged or contain malware. A few reputable websites that host apks and are not known for modifying apps and claim to MD5 checksum the apps are:
```

Comment:
Another question: do they offer SHA-sums? because md5sums are ... easy to collide on a phone :D